### ISSUE TITLE
run_demo_2d_tracklets.py

### ISSUE THREAD

**Issue by mouradkenk** [mpitropov/cadc_devkit#25]
when using run_demo_2d_tracklets.py, 2d_annotations.json files are needed and are missing after download the dataset
could you please share 2d_annotations.json with me?
Thank you in advance

**Comment by mpitropov** [comment]
We only have 3D annotations released.

We had Scale AI also create 2D annotations but as of right now I have not gone through them to check their accuracy and release them.

**Comment by kkannan8291** [comment]
@mpitropov Is there a tentative timeline as to when the 2d_annotations will be released for the CADC dataset?

**Comment by mpitropov** [comment]
@kkannan8291 No, there is not a timeline at the moment. I'm working on my thesis and a paper right now that must be completed by the end of September.
